sites/public/pages/account/application.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useEffect, useState, useContext } from "react"
+import {
+  ApiClientContext,
+  RequireLogin,
+  t,
+  UserContext,
+  FormCard,
+} from "@bloom-housing/ui-components"
+import Link from "next/link"
+import FormSummaryDetails from "../../src/forms/applications/FormSummaryDetails"
+import FormsLayout from "../../layouts/forms"
+import { Application, Listing } from "@bloom-housing/backend-core/types"
+import { useRouter } from "next/router"
+import moment from "moment"
+
+const dateSubmitted = (submissionDate: Date) => {
+  if (!submissionDate) return null
+  const formattedSubmissionDate = moment(new Date(submissionDate)).utc()
+  const month = formattedSubmissionDate.format("MMMM")
+  const day = formattedSubmissionDate.format("DD")
+  const year = formattedSubmissionDate.format("YYYY")
+
+  return `${month} ${day}, ${year}`
+}
+
+export default () => {
+  const router = useRouter()
+  const applicationId = router.query.id as string
+  const { applicationsService, listingsService } = useContext(ApiClientContext)
+  const { profile } = useContext(UserContext)
+  const [application, setApplication] = useState<Application>()
+  const [listing, setListing] = useState<Listing>()
+  const [unauthorized, setUnauthorized] = useState(false)
+  const [noApplication, setNoApplication] = useState(false)
```

Comment:
These 403/404 state hooks are neat. I wonder if we could make a standard hook that would provide this and set the appropriate state on either 404 or 403, so we could use it on other pages. This might be good to review as well if we move towards using SWR everywhere: https://swr.vercel.app/docs/error-handling

Reply:
Refactored this to be a little cleaner, but would hope we could accomplish this in another ticket?